src/components/Charts/BoxedAnnotation.tsx

```diff
@@ -33,6 +33,8 @@ export const BoxedAnnotation = ({
         x={left - padding}
         width={Math.abs(width + 2 * padding)}
         height={Math.abs(height + 2 * padding)}
+        rx={3}
+        ry={3}
```

Comment:
**Note**: According to [CSS Tricks](https://css-tricks.com/svg-properties-and-css/), we should be able to set`rx` and `ry` in CSS, but they are ignored in Chrome iOS. Adding them as attributes solves the issue.